### ISSUE TITLE
I can't connect to extensions store in manjaro.

### ISSUE THREAD

**Issue by OrangeX4** [microsoft/vscode#128974]
Issue Type: <b>Bug</b>

Access to XMLHttpRequest at 'https://open-vsx.org/vscode/gallery/extensionquery' from origin 'vscode-file://vscode-app' has been blocked by CORS policy: Request header field x-market-user-id is not allowed by Access-Control-Allow-Headers in preflight response.

VS Code version: Code - OSS 1.58.0 (2d23c42a936db1c7b3b06f918cde29561cc47cd6, 2021-07-12T23:51:36.303Z)
OS version: Linux x64 5.10.49-1-MANJARO
Restricted Mode: Yes

<details>
<summary>System Info</summary>

|Item|Value|
|---|---|
|CPUs|AMD Ryzen 5 3550H with Radeon Vega Mobile Gfx (8 x 3690)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: disabled_off<br>opengl: enabled_on<br>rasterization: disabled_software<br>skia_renderer: enabled_on<br>video_decode: disabled_software<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|1, 1, 1|
|Memory (System)|14.64GB (10.29GB free)|
|Process Argv|--unity-launch|
|Screen Reader|no|
|VM|0%|
|DESKTOP_SESSION|plasma|
|XDG_CURRENT_DESKTOP|KDE|
|XDG_SESSION_DESKTOP|KDE|
|XDG_SESSION_TYPE|x11|
</details>Extensions: none

**Comment by lnwor** [comment]
I have the same issue on arch

**Comment by tamuratak** [comment]
duplicate of #128583

**Comment by OrangeX4** [comment]
I have fixed the problem.

There are two versions of code, I install vscode instead of code-oss and fix it.

By `sudo pacman -S visual-studio-code-bin`.
